packages/Utils/test/CMakeLists.txt

```diff
@@ -1,5 +1,13 @@
 
 TRIBITS_ADD_EXECUTABLE_AND_TEST(
+  Miscellaneous_test
+  SOURCES tstMiscellaneous.cpp ${TEUCHOS_STD_UNIT_TEST_MAIN}
+  COMM serial mpi
```

Comment:
I will remove the line and squash before we merge but it does not resolve the issue that this tests should only be serial.